tsdb/block_test.go

```diff
@@ -727,7 +727,8 @@ func createHeadWithOOOSamples(tb testing.TB, w *wlog.WL, series []storage.Series
 	require.GreaterOrEqual(tb, actOOOAppended, float64(oooSamplesAppended-len(series)))
 	require.LessOrEqual(tb, actOOOAppended, float64(oooSamplesAppended))
 
-	require.Equal(tb, float64(totalSamples), prom_testutil.ToFloat64(head.metrics.samplesAppended))
+	// This has been broken by splitting samples into floats and histograms.
```

Comment:
The comment is unclear, so I wanted to see where it failed, uncommented it, and all tests still pass 🤔 .